packages/paste-icons/tools/actions/convertNewAction.js

```diff
@@ -14,15 +14,21 @@ const {reactIconTemplate} = require('../templates/reactIconTemplate');
 const {writeToFile} = require('../../../../tools/utils/writeToFile');
 
 // Converts raw svg to react component
-function performFileConversion(fileName, outputPath, options) {
+async function performFileConversion(fileName, outputPath, options) {
```

Comment:
question: does this need to be async? line 21's async function should be enough to wrap the await on line 27 maybe

Reply:
I think you're right! I changed this when I was still trying to debug the svg-to-react issues.